### ISSUE TITLE
Issues when installing Launchers

### ISSUE THREAD

**Issue by Vegemalation** [moraroy/NonSteamLaunchersDecky#10]
Hi there, I installed the decky version of the nonsteamlaunchers through the testing store channel. After selecting "Game launchers", ticking the launchers that I want installed and clicking the big "install" button below. After all installing correctly and showing the notification confirming, once I return to the steam library there aren't any launchers in the "non steam" tab :( ive this process a few times, including a start fresh and restarting the deck to no avail. I was just wondering if you'd be able to give me a hand!

All the best

**Comment by moraroy** [comment]
@Vegemalation this has been fixed in the latest version I believe

**Comment by codecivet** [comment]
Exact same thing for me on the latest version in the Decky store (1.0.1-665c29d). I have been trying unsuccessfully to install Epic and Ubisoft launchers. I've tried wiping, uninstalling, rebooting, everything I can think of, but no launchers showing up.

**Comment by moraroy** [comment]
@codecivet what about the one in the testing store?

**Comment by codecivet** [comment]
> @codecivet what about the one in the testing store?

That's the one I was using. I did get it to work, but unfortunately I tried a lot of things at once and I'm not sure which one fixed it. I installed the NonSteamLaunchers in desktop mode (ie not in Decky) and tried to add Ubisoft from there. It still didn't show up when I booted into Steam mode, so I tried installing from Decky again. Some time after that, but not immediately, Ubisoft showed up.
